### PR TITLE
Add UUID to insights facets that are missing it

### PR DESCRIPTION
**Pull request by ShimShtein**

### CODE REVIEW

File: lib/inventory_sync/async/inventory_hosts_sync.rb [hunk 1]
```diff
@@ -20,14 +20,19 @@ def create_facets
...
           {
             host_id: host_id,
             uuid: uuid,
           }
         end
         InsightsFacet.create(missing_facets)
+        missing_uuids = existing_facets.select { |host_id, uuid| uuid.empty? }.map(&:first)
+
+        missing_uuids.each do |host_id|
+          InsightsFacet.where(host_id: host_id).update_all(uuid: uuids_hash[host_id])
```
Comment (adamruzicka) [inline lib/inventory_sync/async/inventory_hosts_sync.rb:34]: How about something like this this?

```suggestion
        missing_uuids = existing_facets.select { |host_id, uuid| uuid.empty? }.each do |host_id, uuid|
          InsightsFacet.where(host_id: host_id).update_all(uuid: uuid)
```

**Review by adamruzicka — COMMENTED** [review]
One suggestion, but it might be down to personal taste. Otherwise looks good

**Comment by ShimShtein** [comment]
@adamruzicka Done!

**Review by adamruzicka — APPROVED** [review]

**Review by laviro — APPROVED** [review]
Thanks @ShimShtein !

**Pull request merged**